exercises/practice/knapsack/knapsack.R

```diff
@@ -0,0 +1,3 @@
+maximum_value <- function(max_wt, items) {
```

Comment:
Maybe make this slightly more verbose?

```suggestion
maximum_value <- function(max_weight, items) {
```